src/olympia/blocklist/views.py

```diff
@@ -132,6 +156,46 @@ def between(ver, min, max):
 
 
 @non_atomic_requests
+def blocklist_json(request):
+    key = 'blocklist:json'
+    cache.add('blocklist:keyversion', 1)
+    version = cache.get('blocklist:keyversion')
```

Comment:
Right. But it returns whether or not the key existed - if it did not, then the value you gave (`1`) is going to be the value you get in the next call, unless it was modified by something else in between, but I don't think we care much about that. Not a huge deal anyway, you can leave it.